From jrm8005@gmail.com  Wed Oct 29 11:06:19 2008
From: "J.R. Mauro" <jrm8005@gmail.com>
Date: Tue, 28 Oct 2008 19:00:56 -0400
Subject: Staging: sxg: break the build in a cleaner way when !x86
To: Greg KH <greg@kroah.com>
Message-ID: <20081028230056.GD1267@localhost>
Content-Disposition: inline

Remove ugly 'Stop Comilation;' statement in sxghif, replace with an
error macro.

This should never be hit as we are only building for x86 boxes at the
moment.

Signed-off by: J.R. Mauro <jrm8005@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/sxg/sxghif.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/sxg/sxghif.h
+++ b/drivers/staging/sxg/sxghif.h
@@ -854,5 +854,5 @@ struct SXG_SCATTER_GATHER {
 #define SXG_SGL_BUFFER(_SxgSgl)		NULL
 #define SXG_SGL_BUF_SIZE			0
 #else
-Stop Compilation;
+#error staging: sxg: driver is for X86 only!
 #endif
